README.md

```diff
@@ -1512,6 +1512,53 @@ would enable the use of a `@fold` on the `adjacent_animal` vertex field of `Foo`
 }
 ```
 
+## Schema Generation
+To generate a GraphQL schema, we first map a database schema into an intermediate representation:
+the `SchemaGraph`.
```

Comment:
I'd define the schema graph not as an IR, but as the python object representing the schema in the most convenient way for introspection, and that not all this information can be encoded in a graphql schema (indexes for example).